src/components/article.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from 'react';
+import { Link, useParams } from 'react-router-dom';
+const axios = require('axios');
+
+const Article = () => {
+	const [AvailableArticle, setAvailableArticle] = useState([]);
+	const [isLoading, setIsLoading] = useState([true]);
+	const { article_id } = useParams();
+
+	useEffect(() => {
```

Comment:
This is absolutely spot on, but would be nice if it were extracted out into a separate file along with any other api calls.